sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupClientBuilder.java

```diff
@@ -211,14 +224,17 @@ public KeyVaultBackupClientBuilder vaultUrl(String vaultUrl) {
     }
 
     /**
-     * Sets the credential to use when authenticating HTTP requests.
+     * Sets the {@link TokenCredential} used to authorize requests sent to the service. Refer to the Azure SDK for Java
+     * <a href="https://aka.ms/azsdk/java/docs/identity">identity and authentication</a>
+     * documentation for more details on proper usage of the {@link TokenCredential} type.
      *
-     * @param credential The credential to use for authenticating HTTP requests.
+     * @param credential {@link TokenCredential} used to authorize requests sent to the service.
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores the value set in this method. We should probably add a note like you did in other methods on this same class.